tests/test_pyglmnet.py

```diff
@@ -207,12 +210,17 @@ def test_glmcv():
             X_train = scaler.fit_transform(X_train)
             glm.fit(X_train, y_train)
 
-            beta_ = glm.beta_
-            assert_allclose(beta, beta_, atol=0.5)  # check fit
+            betas_.append(glm.beta_)
+            assert_allclose(beta, betas_[-1], atol=0.5)  # check fit
 
             y_pred = glm.predict(scaler.transform(X_train))
             assert_equal(y_pred.shape[0], X_train.shape[0])
 
+        # compare all solvers pairwise to make sure they're close
+        for i, first_beta in enumerate(betas_[:-1]):
+            for second_beta in betas_[i+1:]:
+                assert_allclose(first_beta, second_beta, atol=0.5)
```

Comment:
@pavanramkumar the tolerance looks really bad. I think that's the reason it wasn't merged